lib/tools/adb-commands.js

```diff
@@ -344,6 +344,36 @@ methods.toggleGPSLocationProvider = async function (enabled) {
   await this.setSetting('secure', 'location_providers_allowed', `${enabled ? "+" : "-"}gps`);
 };
 
+/**
+ * Permit access to all non-SDK APIs, but keep warnings in the log.
+ * https://developer.android.com/preview/restrictions-non-sdk-interfaces
+ *
+ * Commands don't lead crash or something, thus simply this method calls putting all setting.
+ *
+ * @param {number} value - The API enforcement policy.
+ *     0: Disable non-SDK API usage detection. This will also disable logging, and also break the strict mode API,
+ *        detectNonSdkApiUsage(). Not recommended.
+ *     1: "Just warn" - permit access to all non-SDK APIs, but keep warnings in the log.
+ *        The strict mode API will keep working.
+ *     2: Disallow usage of dark grey and black listed APIs.
+ *     3: Disallow usage of blacklisted APIs, but allow usage of dark grey listed APIs.
+ */
+methods.setHiddenApiPolicy = async function (value) {
+  await this.setSetting('global', 'hidden_api_policy_pre_p_apps', value.toString());
```

Comment:
`toString` is redundant. Also the parameter type could be either string or number (`'1'` and `1` will work equally)